src/parser/Parser.ts

```diff
@@ -1187,6 +1188,36 @@ export class Parser {
         return annotation;
     }
 
+    private ternaryExpression(test?: Expression): TernaryExpression {
+        this.warnIfNotBrighterScriptMode('ternary operator');
+        if (!test) {
+            test = this.expression();
+        }
+        const questionMarkToken = this.advance();
```

Comment:
Is/should a newline allowed here?
Maybe some wrapping at places can be safely allowed - and if yes/no it might be good to document.

Reply:
Good point. I don't see any reason to prevent newlines within ternary statements. In brs, you can use the question mark in place of print, so it might be good to require the condition and the question mark to be in the same line. Thoughts?

The rest can definitely be flexible to use newlines at will. 

So something like this would be supported. 

```
someVar = true = true ?
                      getTrueValue():

                       getFalseValue()
```

Reply:
Any other expression brine already supported by brs elsewhere would be automatically supported too

Reply:
Basically yes, `?` can't be on a newline but otherwise the rest could be wrapped.

Reply:
Done. I also added support for mixing comments throughout (but they aren't included in the transpiled output because there's not much benefit in retaining them IMO. (Let me know if you disagree). 

Here are the tests for the various ways you can use 

```
    it('supports multi-line and comments', () => {
        expectZeroDiagnostics(parseBs(`result = true ? \n"zombie"\n: \ntrue`));
        expectZeroDiagnostics(parseBs(`result = true ? "zombie"\n: \ntrue`));
        expectZeroDiagnostics(parseBs(`result = true ? \n"zombie": \ntrue`));
        expectZeroDiagnostics(parseBs(`result = true ? \n"zombie"\n: true`));
        expectZeroDiagnostics(parseBs(`result = true ? "zombie"\n: \ntrue`));
        expectZeroDiagnostics(parseBs(`result = true ? "zombie": \ntrue`));
        expectZeroDiagnostics(parseBs(`result = true ? \n\n\n"zombie": \n\n\n\ntrue`));
        //with comments
        expectZeroDiagnostics(parseBs(`result = true ?'comment\n"zombie"'comment\n:'comment\nntrue`));
    });
    ```

Reply:
Cool